exe-common/src/sync.rs

```diff
@@ -173,6 +171,28 @@ pub fn net_sync<A: Api>(
     Ok(())
 }
 
+pub fn net_sync<A: Api>(
```

Comment:
can you document a bit this function ? There are lot of thing we need to remember when using this function, one of them being this function will use `thread::sleep` for a given amount of time when using `hermes`.